lib/aws_sdk_rails.rb

```diff
@@ -1,34 +1,31 @@
+# frozen_string_literal: true
+
 require_relative 'aws/rails/mailer'
 
 module Aws
+  # Use the Rails namespace.
   module Rails
-
     # @api private
     class Railtie < ::Rails::Railtie
-      initializer "aws-sdk-rails.initialize", before: :load_config_initializers do |app|
+      initializer 'aws_sdk_rails.initialize',
+                  before: :load_config_initializers do
         # Initialization Actions
-        Aws::Rails.use_rails_encrypted_credentials
         Aws::Rails.add_action_mailer_delivery_method
         Aws::Rails.log_to_rails_logger
+        Aws::Rails.use_rails_encrypted_credentials
```

Comment:
Does this need to go before `add_action_mailer_delivery_method`?  Or does the mailer use the newly-added credentials regardless of order?  Should there be another test to verify this?

Reply:
I get in the habit of alphabetizing methods and method calls - but that is a good point. This *should* be first, otherwise, the client can be constructed using the provider chain instead of encrypted credentials. I will fix this.